src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -118,13 +119,78 @@
 (def default-allow-auth-extensions
   false)
 
+(def default-cadir
+  "/etc/puppetlabs/puppetserver/ca")
+
+(defn default-cacert
+  [cadir]
+  (str cadir "/ca_crt.pem"))
+
+(defn default-cacrl
+  [cadir]
+  (str cadir "/ca_crl.pem"))
+
+(defn default-cakey
+  [cadir]
+  (str cadir "/ca_key.pem"))
+
+;; TODO how do we get certname here?
+(def default-caname
+  "Puppet CA: $certname")
+
+;; TODO where should this actually live?
+(defn default-autosign
+  [cadir]
+  (str cadir "/autosign.conf"))
```

Comment:
This currently lives in puppet's confdir. Is it something we want to move, or not?